test/tests/functional/pbs_multiple_execjob_launch_hook.py

```diff
@@ -63,6 +64,7 @@ class TestExecJobHook(TestFunctional):
 else:
     pbs.logmsg(pbs.LOG_DEBUG,
         "Modified progname value got updated from launch1")
+pbs.logmsg(pbs.LOG_DEBUG, "environment var from execjob_hook2 is %s" %(e.env))
```

Comment:
Indentation is incorrect. Please run pep8 on your python files and check file permissions.

Reply:
Perhaps it's an artifact of they way GitHub displays the diff. If pep8 is happy, I'm fine with it.